float64     target_heading
float64     distance
float64     waypoint
